src/olympia/addons/models.py

```diff
@@ -374,6 +374,14 @@ class Addon(OnChangeMixin, ModelBase):
 
     class Meta:
         db_table = 'addons'
+        index_together = [
+            ['weekly_downloads', 'type'],
+            ['created', 'type'],
+            ['bayesian_rating', 'type'],
+            ['last_updated', 'type'],
+            ['average_daily_users', 'type'],
```

Comment:
I thought so as well, but django docs example is a list: https://docs.djangoproject.com/en/1.11/ref/models/options/#index-together